ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -47,6 +49,9 @@ function WorkloadCvesOverviewPage() {
 
     const pagination = useURLPagination(20);
 
+    const [defaultWatchedImageName, setDefaultWatchedImageName] = useState('');
```

Comment:
@dvail Is this just a placeholder for now?

Reply:
Yep, this is going to be used as the default for the image name input in a later PR. I wanted to have the structure in place to minimize moving things around once the modal is more filled out.